src/sentry/static/sentry/app/components/button/index.jsx

```diff
@@ -88,7 +88,8 @@ class Button extends React.Component {
     } = this.props;
 
     // For `aria-label`
-    let screenReaderLabel = label || typeof children === 'string' ? children : undefined;
```

Comment:
This is actually:

`(label || typeof children === 'string') ? children : undefined;`